packages/playground/windows/playground/MainPage.cpp

```diff
@@ -73,6 +73,24 @@ void MainPage::OnLoadClick(
     host.InstanceSettings().DebugHost(m_bundlerHostname);
   }
 
+  host.InstanceSettings().InstanceCreated(
+      [wkThis = get_weak()](auto sender, winrt::Microsoft::ReactNative::InstanceCreatedEventArgs args) {
+      if (auto strongThis = wkThis.get()) {
+          args.Context().UIDispatcher().Post([wkThis, context = args.Context()]() {
```

Comment:
Do we still get a valid Context if "args.Failed()"?

Reply:
Failed is part of loaded, not created.  The creation event wouldn't fire if creation failed.